admin/make-tarball

```diff
@@ -35,11 +35,9 @@ if [[ ! -d .git && ! -f .git ]]; then
     exit 1
 fi
 
-VERSION=$( cylc -v )
-if [[ $VERSION = *-dirty ]]; then
-    echo "The work tree is not clean: ABORTING" >&2
-    exit 1
-fi
+THIS_DIR=$( cd $( dirname $0 ); pwd; )
```

Comment:
Just curious, compared to plain old `THIS_DIR=$(dirname $0)` this construct makes the path absolute if the script is invoked by a relative path - but I don't think that matters in this context.  Is there some other advantage?

Reply:
I agree that that removing the "-dirty" abort is good, now that this script is no longer used to make official releases, and "-dirty" appears in the tarball name, but you just need to update (delete) the last paragraph of the command usage string accordingly.

Reply:
Re the path thing, it is typically what I do to prevent bad corner cases.

Reply:
Have updated the usage string.